Mathlib/Combinatorics/Catalan.lean

```diff
@@ -192,7 +192,7 @@ theorem treesOfNumNodesEq_succ (n : ℕ) :
 theorem mem_treesOfNumNodesEq {x : Tree Unit} {n : ℕ} :
     x ∈ treesOfNumNodesEq n ↔ x.numNodes = n := by
   induction x using Tree.unitRecOn generalizing n <;> cases n <;>
-    simp [treesOfNumNodesEq_succ, Nat.succ_eq_add_one, *]
+    simp [treesOfNumNodesEq_succ, Nat.succ_eq_add_one, mem_antidiagonal, *]
```

Comment:
I pushed a change that made `mem_antidiagonal` a `simp` lemma (to match the `Nat.mem_antidiagonal` it replaced), so you should be able to remove all the changes of this form

Reply:
what is simp[..., *] ?

Reply:
I don't know how to detect them, anyway.